test/built-ins/Map/prototype/get/returns-value.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
```

Comment:
This looks like a less-complete version of test/built-ins/Map/prototype/get/returns-value-different-key-types.js . I think it's safe to remove it.